spec/requests/users_spec.rb

```diff
@@ -1,38 +1,26 @@
 require 'rails_helper'
 
-RSpec.describe UsersController, type: :request do
-  describe 'GET /index' do
-    it 'returns a success response' do
-      get '/users/'
-      expect(response).to be_successful
+RSpec.describe User, type: :model do
+  describe 'validations for User model' do
+    before(:each) do
+      @user = User.new(name: 'Tom', photo: 'image.png', bio: 'Teacher from Mexico', post_counter: 0)
     end
 
-    it 'renders the index template of views/users' do
-      get '/users/'
-      expect(response).to render_template(:index)
-    end
-  end
-  it 'includes the correct placeholder text' do
-    get '/users/'
-    result = response.body
-    expect(result).to include('Users List')
-  end
+    before { @user.save }
 
-  describe 'GET /show' do
-    it 'returns a success response' do
-      get '/users/745/'
-      expect(response).to have_http_status(200)
+    it 'if there is name' do
```

Comment:
* As stated in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=corresponding%20wireframe%20view%3A-,Integration%20specs,When%20I%20click%20on%20a%20post%2C%20it%20redirects%20me,-to%20that%20post%27s) you are to Use Capybara to write integration tests for each view in your project which are the `user index page`, `user show page`, `user post index page` and `post show page` kindly write tests that covers all these scenarios highlighted in the screenshot below 👇  
![image](https://user-images.githubusercontent.com/52497246/233134757-1d5958e9-6e25-48e1-9beb-ed1c9b8e35a1.png)